### DISCUSSION TITLE
Add needed feature - Multiply Cordinators or separate instans/list

### DISCUSSION THREAD

**Discussion by JanKW123** [Koenkk/zigbee2mqtt#25354] [General]
It would be fantastic if you could easily add more cordinators, for larger houses and more buildings. 

I found this hack on youtube: 
https://www.youtube.com/watch?v=ZBbl0I-Zpuk 
it works, but hard to use, as devices from the different cordinators mix in the same list etc.

**Comment by lux73** [comment]
absolutely unnecessary. Focus on a strong, stable Zigbee Mesh
